language/src/cli.yml

```diff
@@ -4,7 +4,7 @@ author: Denis Merigoux <denis.merigoux@inria.fr>
 about: Typechecker and compiler for the Hacspec subset of Rust
 args:
     - INPUTS:
-        help: Sets the input files to use
+        help: Path to the crate to analyze
```

Comment:
So I tried to fiddle with the arguments after doing a `cargo install -- path ./` and it's not clear to me. First, if I try to set multiple inputs I get an error so you should set èmultiple` to `false` in line 10. But then what i the "path to the crate" ? Is in the path to the directory containing the `Cargo.toml` of the crate ? Is it a relative path or absolute ? 

My intuition is that we would use the typechecker by `cd` into the crate man directory (where `Cargo.toml` is) without any arguments and then just do `cargo hacspec`.

Reply:
I cleaned up usage. I'll merge and we need to follow up anyway for documentation. But I think it should be clearer now. To test locally do a `cargo install --path . --force`. Then you can do `cargo +nightly hacspec hacspec-chacha20` in the according crate (or workspace), or `cargo +nightly hacspec -o Hacspec.Chacha20.fst hacspec-chacha20`. When we're happy with the usage and the docs are good enough I publish the crate as 0.2 (0.1 is published already).